autode/smiles/builder.py

```diff
@@ -718,8 +725,8 @@ def _queue_double_bond_dihedral(self, bond):
         nbrs_x = [idx for idx in self.atoms[idx_x].neighbours if idx != idx_y]
         nbrs_y = [idx for idx in self.atoms[idx_y].neighbours if idx != idx_x]
 
-        if len(nbrs_x) < 2 or len(nbrs_y) < 2:
-            logger.info('Had a double bond containing an atom with < 2 '
+        if len(nbrs_x) < 1 or len(nbrs_y) < 1:
```

Comment:
i was having this discussion with my tech lead today. Its clearer to have == 0 here instead of < 1